tfhe/src/core_crypto/fft_impl/fft64/math/fft/mod.rs

```diff
@@ -404,6 +404,36 @@ impl<'a> FftView<'a> {
         self.forward_with_conv(fourier, standard, convert_forward_integer, stack)
     }
 
+    #[must_use]
+    pub fn incomplete_monomial_forward_as_integer(
```

Comment:
can we maybe add the management of the X^N + 1 polynomial ring and have a "complete" monomial forward ? using the polynomial size from fourier ?